### PR TITLE
[Squid][SMB][Mult-Connection]Add multi-connection support

### PR DESCRIPTION
**Pull request by MohitBis**
[Squid][SMB][Mult-Connection]Add multi-connection support

### CODE REVIEW

**Comment by MohitBis** [comment]
Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-QDETA0/Samab_with_client_mutli-connection_0.log

File: suites/squid/smb/tier-2-smb-multi-connections.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        node: node4
+        install_packages:
+          - ceph-common
+          - samba-client
+      copy_admin_keyring: true
+      destroy-cluster: false
+      abort-on-fail: true
+
+  - test:
+      name: Samab with client mutli-connection
```
Comment (pranavprakash20) [inline suites/squid/smb/tier-2-smb-multi-connections.yaml:80]: small typo; "samba"

File: suites/squid/smb/tier-2-smb-multi-connections.yaml [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      desc: Load Testing - Samab with client mutli-connection
```
Comment (pranavprakash20) [inline suites/squid/smb/tier-2-smb-multi-connections.yaml:81]: small typo; "samba"

File: suites/squid/smb/tier-2-smb-multi-connections.yaml (hunk 1 above)
Comment (MohitBis) [reply suites/squid/smb/tier-2-smb-multi-connections.yaml]: Thanks @pranavprakash20 
Updated

File: suites/squid/smb/tier-2-smb-multi-connections.yaml (hunk 2 above)
Comment (MohitBis) [reply suites/squid/smb/tier-2-smb-multi-connections.yaml]: Thanks @pranavprakash20 

Updated

File: suites/squid/smb/tier-2-smb-multi-connections.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (MohitBis) [reply suites/squid/smb/tier-2-smb-multi-connections.yaml]: Thanks @adityaramteke 

Updated

**Review by adityaramteke — APPROVED** [review]

**Review by pranavprakash20 — APPROVED** [review]
Lgtm

**Review by vdas-redhat — APPROVED** [review]
LGTM

**Pull request merged**